### ISSUE TITLE
The model file is not exist

### ISSUE THREAD

**Issue by jingwenli0312** [yd-kwon/MatNet#1]
![image](https://user-images.githubusercontent.com/62284365/144698094-5b14a4a6-7bb3-4fb7-a21e-1ab44e04e31f.png)

Hi, it seems that the trained model saved in results/saved_atsp_model is not available now. I tried to get the model from git lfs, but it shows the model does not exist on the server. Could you please solve this problem? Thanks very much.

**Comment by yd-kwon** [comment]
Thank you for pointing out this error. I have uploaded a new model file just now.
(I have not tested this model with the codes in github. Hopefully it will work, but let me know if it does not!)

Once again, thanks!

**Comment by yd-kwon** [comment]
I have tested running the test code for ATSP MatNet (N=20 case).
It works well with the uploaded model file. Below is the generated log.txt :

[2021-12-14 11:57:24] test.py(139) : DEBUG_MODE: False
[2021-12-14 11:57:24] test.py(140) : env_params{'node_cnt': 20, 'problem_gen_params': {'int_min': 0, 'int_max': 1000000, 'scaler': 1000000}, 'pomo_size': 20}
[2021-12-14 11:57:24] test.py(140) : model_params{'embedding_dim': 256, 'sqrt_embedding_dim': 16.0, 'encoder_layer_num': 5, 'qkv_dim': 16, 'sqrt_qkv_dim': 4.0, 'head_num': 16, 'logit_clipping': 10, 'ff_hidden_dim': 512, 'ms_hidden_dim': 16, 'ms_layer1_init': 0.7071067811865476, 'ms_layer2_init': 0.25, 'eval_type': 'softmax', 'one_hot_seed_cnt': 20}
[2021-12-14 11:57:24] test.py(140) : tester_params{'use_cuda': True, 'cuda_device_num': 0, 'model_load': {'path': './result/saved atsp20_model', 'epoch': 5000}, 'saved_problem_folder': '../data/n20', 'saved_problem_filename': 'problem_20_0_1000000_{}.atsp', 'file_count': 10000, 'test_batch_size': 100, 'augmentation_enable': True, 'aug_factor': 128, 'aug_batch_size': 100}
[2021-12-14 11:57:24] test.py(140) : logger_params{'log_file': {'desc': 'atsp_matnet_test', 'filename': 'log.txt', 'filepath': './result/20211214_115723_atsp_matnet_test'}}
[2021-12-14 11:57:31] ATSPTester.py(82) :  *** Loading Saved Problems *** 
[2021-12-14 11:57:41] ATSPTester.py(94) : Done. 
[2021-12-14 11:57:44] ATSPTester.py(122) : episode 100/10000, Elapsed[0.04m], Remain[3.98m], score:1.518, aug_score:1.512
[2021-12-14 11:57:46] ATSPTester.py(122) : episode 200/10000, Elapsed[0.08m], Remain[3.92m], score:1.594, aug_score:1.585
...
[2021-12-14 12:01:38] ATSPTester.py(122) : episode 9900/10000, Elapsed[3.94m], Remain[0.04m], score:1.571, aug_score:1.560
[2021-12-14 12:01:40] ATSPTester.py(122) : episode 10000/10000, Elapsed[3.98m], Remain[0.00m], score:1.538, aug_score:1.533
[2021-12-14 12:01:40] ATSPTester.py(128) :  *** Test Done *** 
[2021-12-14 12:01:40] ATSPTester.py(129) :  NO-AUG SCORE: 1.5468 
[2021-12-14 12:01:40] ATSPTester.py(130) :  AUGMENTATION SCORE: 1.5397
